packages/web/src/components/gcds-link/i18n/i18n.js

```diff
@@ -0,0 +1,16 @@
+const I18N = {
+  en: {
+    external: 'Opens in a new tab.',
+    phone: 'May open your phone app. [copy edit needed]',
+    download: 'May download file to your device. [copy edit needed]',
+    email: 'May open your email client. [copy edit needed]',
+  },
+  fr: {
+    external: "S'ouvre dans un nouvel onglet.",
+    phone: 'Phone number [fr & copy edit needed]',
+    download: 'Download [fr & copy edit needed]',
+    email: 'Email [fr & copy edit needed]',
+  },
+};
+
+export default I18N;
```

Comment:
@SmartMouthWords Hi Amy, I could use some help with the language for this. A bit of context: these are the labels for the icons when a link is either a download, an email, or a phone number.
- A download link will download the file to the user's device if the URL is of the same origin. If it's not, it simply opens in the same tab.
- A `mailto:` or email link will attempt to open the user's email client on their device
- A `tel:` link will attempt to open the user's phone app
- Both `mailto:` and `tel:` link actions can change depending on the user's setting, and we have no way of forcing it upon them.

Reply:
> Where are all the places this content will live?

These will be the labels for screen readers (`aria-label`). 
<img width="430" alt="Screenshot 2023-10-25 at 4 32 12 PM" src="https://github.com/cds-snc/gcds-components/assets/916044/00353403-5b19-424d-8b74-c0ff35e4be2e">

> Do you want me to look at the parts labelled copy edit? Or at the whole page?

The copy edits would be great!

> "This is a download link [...] with type indicated", you might leave off "with type indicated". This isn't stated for the others, like phone so may not need to be here. If you think the download icon is less recognizable (probably is), you could add a line about it but then you'd probably want to do the same for the external link icon.


I think I need further clarity on what you mean, let's chat about this soon!

> Could the text "Referrerpolicy" be replaced with "policy reference"?

No, that is an HTML attribute set by W3 standards. Unless I'm missing some context?

> For the labels:
>
> For external, is it always a new tab, could it be a window or something else?
For the email one, you might use "default email" or "default email app" instead of "email client".
Recommend instead of using "may" communicating what will happen in what circumstances and potentially removing "your". For example, instead of "May open your phone app", you could consider stating:
"Will open the phone app that's connected" or
"Will open the phone app. If no phone app is connected, link will be disabled." or
"Will open the phone app or prompt to copy the number"
(Not sure how this will work so included a few examples based on guesses.)

Currently, when this was implemented as a button link, that's the text we have (for external links, it says opens in a new tab). It could open in a new tab or window (from [Mozilla a tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target): usually a new tab, but users can configure browsers to open a new window instead.)

Reply:
Thanks for the discussion earlier this week @daine.  
 
 
    external: 'Opens destination in a new tab.',
    phone: 'Attempts to open a phone app.',
    download: 'Attempts to download a file to the device.',
    email: 'Opens new message in email reader.',
    email: 'Opens new message in email reader.',
OR